source/developers-guide/code-snippets/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Code snippet collection
+github_link: developers-guide/code-snippets/index.md
+shopware_version: 5.2.0
+indexed: true
+---
+
+This article will provide some useful code snippets that can help you when developing Shopware plugins. Please feel free to add additional snippets via GitHub!
+
+## Read  config from a different shop than the current one
+
+```php
+/** @var \Shopware_Components_Config $config */
+$config = \Shopware()->Container()->get('config');
+$config->setShop($shop);
```

Comment:
I think this should be avoided in normal shopware stacks, as it will change the shop context and will affect any code running afterwards and accessing shop configurationss, even templates.

I guess as a workaround the config object could be cloned - otherwise the config reader should be refactored in a way, that it operates stateless.